ProjFS.Mac/PrjFSKext/public/FsidInode.h

```diff
@@ -2,6 +2,7 @@
 #define FsidInode_h
 
 #include <sys/_types/_fsid_t.h>
+#include <stdint.h>
```

Comment:
Why did we need to add this include?

Reply:
It was complaining about `uint64_t` not being defined when included from the unit tests, if I remember correctly.

Reply:
I've just checked, and yes, if you remove that `#include`, it complains about `uint64_t inode;` a few lines down when building the unit tests.